overrides/partials/social.html

```diff
@@ -0,0 +1,21 @@
+<!-- Social links in footer -->
+{% if config.extra.social %}
+  <div class="md-footer-social">
+    {% for social in config.extra.social %}
+      {% set title = social.name %}
+      {% if not title and "//" in social.link %}
+        {% set _,url = social.link.split("//") %}
+        {% set title = url.split("/")[0] %}
+      {% endif %}
+      <a
+        href="{{ social.link }}"
+        target="_blank" rel="noopener"
+        title="{{ title | e }}"
+        class="md-footer-social__link"
+      > 
+        {% include ".icons/" ~ social.icon ~ ".svg" %}
+        <div class="md-footer-social__name">{{social.name}}</div>
```

Comment:
Modifying the theme templates is super simple. The file has just to be in the right place, and it will automatically replace its original: https://squidfunk.github.io/mkdocs-material/customization/

I copied the template for the social icons & just added the name + had to modify the css a bit.